realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -1153,14 +1190,157 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_TableQuery_nativeRemove(
 // isNull and isNotNull
 
 JNIEXPORT void JNICALL Java_io_realm_internal_TableQuery_nativeIsNull(
-    JNIEnv *env, jobject, jlong nativeQueryPtr, jlong columnIndex)
+    JNIEnv *env, jobject, jlong nativeQueryPtr, jlongArray columnIndexes)
 {
+    GET_ARRAY()
+
+    Query* pQuery = Q(nativeQueryPtr);
+
+    try {
+        Table* pTable;
+        jlong columnIndex;
```

Comment:
In general I think we use camelCase for java variables and "underscore_case" (name eludes me) for c++ variables.